XamlControlsGallery/Navigation/NavigationRootPage.xaml

```diff
@@ -71,6 +71,8 @@
             Header=" "
             HeaderTemplate="{StaticResource NavigationViewHeaderTemplate}"
             IsTabStop="False"
+            IsPaneOpen="False"
+            PaneDisplayMode="Auto"
```

Comment:
Auto is the default PaneDisplayMode value. So while this line doesn't hurt anything, it's also not necessary.

Reply:
Cleaned that up in the newest commit.